app/_components/trade-routes/single/response/mobile/BodyMobile.tsx

```diff
@@ -0,0 +1,421 @@
+import React, { useState } from 'react';
```

Comment:
Is there no way we can achieve desktop/mobile layouts without multiple templates? I'm happy to approve this and add a job to look at this later, but long term it's going to be a pain managing multiple templates for different screen sizes

Reply:
Luckily, I think these trade routes will most likely be our most complex elements as far as UI structure. There's just a lot of info to get across no matter the breakpoint. Since there's a lot of unique styling in creating special dividers and relocating data depending on the screen width, I thought this separation made more sense. 

There's other ways to accomplish this for sure. But involving this much Chakra styling just consumes a lot of lines. HTML can be merged but it may get ugly with ternary operators. 

Don't consider these templates. Most of our routes won't present data in tables like this. They'll be more descriptive about a singular thing like a stellar object. They won't need this much styling. I may return to the commodities route and see if that can be simplified. We can look at this more later if necessary.